File: InventoryAppOriginal/app/src/main/java/com/example/brookesellerinventoryapp/Item.java
package com.example.brookesellerinventoryapp;

// item class
public class Item {
    public final long   id;
    public final String name;
    public final String imageUrlOrPath;   // can be null
    public final String sku;
    public final int    quantity;
    public final String upc;              // can be null
    public final String description;      // can be null

    public Item(long id,
                String name,
                String imageUrlOrPath,
                String sku,
                int quantity,
                String upc,
                String description) {
        this.id = id;
        this.name = name;
        this.imageUrlOrPath = imageUrlOrPath;
        this.sku = sku;
        this.quantity = quantity;
        this.upc = upc;
        this.description = description;
    }
}

File: BrookeSellerInventoryApp/app/src/main/kotlin/com/example/brookesellerinventoryapp/Item.kt
package com.example.brookesellerinventoryapp

// item class
class Item(
    @JvmField val id: Long,
    @JvmField val name: String?,
// can be null
    @JvmField val imageUrlOrPath: String?,
    @JvmField val sku: String?,
    @JvmField val quantity: Int,
    // can be null
    @JvmField val upc: String?,
    // can be null
    @JvmField var description: String?
) {
    init {
        this.description = description
    }
}
